Number_types/include/CGAL/NT_converter.h

```diff
@@ -116,6 +122,18 @@ struct NT_converter < Interval_nt<b>, Interval_nt<b> >
     }
 };
 
+template < class NT1, class NT2 >
+struct NT_converter < Quotient<NT1>, Quotient<NT2> >
```

Comment:
It does actually. I thought I had checked with an `Homogeneous_converter<K, K>`, but in that case it actually has a specialization that just forwards everything immediately and doesn't have to go down to the level of `NT_converter`.

I could add a default base template parameter to SFINAE things around, but it's ugly; would you see anything more elegant?

Reply:
All the other specializations of NT_converter come in pairs, for instance `NT_converter<NT1, float>` and `NT_converter<float, float>`, doing the same for Quotient would be the easiest. Using SFINAE to disable this specialization in the equal case would also be nice, and would allow removing several of the *seemingly* redundant specializations, I don't think it is that ugly. C++20 concepts probably allow this without the dummy template parameter, but it is a bit early to require C++20 for CGAL :wink: